### ISSUE TITLE
Analyst exercise may need to be updated

### ISSUE THREAD

**Issue by johnwunder** [STIXProject/stixproject.github.io#235]
Need to read through it to determine if a report should be added.

**Comment by bschmoker** [comment]
Are you suggesting adding a `Report` to contain the threat actor and stuff? 

Suggest keeping that example as basic as possible, and having a specific example for `Report` using a different one

**Comment by johnwunder** [comment]
I mean I'm all for simplicity but it's very clearly a report, we call it a report in that writeup like 5 times, and so it seems like it might be confusing to not have a Report in there.

**Comment by bschmoker** [comment]
I can do a quick draft of this

```
move the title, description, and package_intent from the STIX_Package/STIX_Header into a new report and then add references back to all the constructs.
```

merge with https://github.com/STIXProject/stixproject.github.io/issues/239 ?

**Comment by johnwunder** [comment]
I wouldn't merge w/ #239, like you said simpler is better and so for that idiom I think we want to talk specifically about the report and not add all this other stuff.

**Comment by bschmoker** [comment]
:+1: makes sense

**Comment by bschmoker** [comment]
made PR https://github.com/STIXProject/stixproject.github.io/pull/246

**Comment by johnwunder** [comment]
Closed by dcb77ff
